pylint/interfaces.py

```diff
@@ -26,16 +26,34 @@
 if TYPE_CHECKING:
     from pylint.reporters.ureports.nodes import Section
 
+__all__ = (
+    "IRawChecker",
+    "IAstroidChecker",
+    "ITokenChecker",
+    "IReporter",
+    "IChecker",
+    "Confidence",
+    "NO_INFERENCE",
+    "HIGH",
+    "INFERENCE",
+    "INFERENCE_FAILURE",
+    "UNDEFINED",
+    "CONFIDENCE_LEVELS",
+)
+
 Confidence = namedtuple("Confidence", ["name", "description"])
```

Comment:
I'd want to do an enum here, but it's probably not worth the breaking change. On the other hand maybe addint it to ``__all__`` send the message that it's a public API.

Reply:
Should `Confidence` be added here? Isn't that something we only use internally?

Reply:
Right it's a remnant of the refactor to replace all constant by an enum containing the constants.